src/graphql/operations/leaderboards.ts

```diff
@@ -8,31 +9,35 @@ export default async function (parent, args) {
 
   checkLimits(args, 'leaderboards');
 
+  const ORDER_FIELDS = ['vote_count', 'proposal_count', 'user'];
```

Comment:
I don't think we would ever use `user` in orderby, I think we may use the last vote to sort recently active users

Reply:
We could add `last_vote` and use it internally, but `user` has been added to keep follow sx api

https://api-1.snapshotx.xyz/?explorerURLState=N4IgJg9gxgrgtgUwHYBcQC4QEcYIE4CeABMADpJFEDOADgIZQIAUAlmOkaSAigBboAPIQK4BKEuUqU2kogF9yckHKA

Reply:
i see, got it. 

yea i think better to add last vote and space, but not important for now. feel free to merge

Reply:
- `last_vote` added to the `ORDER BY`
- Add back all other fields (`space`) to the order by whitelist